src/components/language/ethnologue-language/ethnologue-language.service.ts

```diff
@@ -0,0 +1,367 @@
+import {
+  Injectable,
+  InternalServerErrorException as ServerException,
+} from '@nestjs/common';
+import { node, relation } from 'cypher-query-builder';
+import { upperFirst } from 'lodash';
+import { DateTime } from 'luxon';
+import { generate } from 'shortid';
+import { ISession } from '../../../common';
+import {
+  ConfigService,
+  DatabaseService,
+  ILogger,
+  Logger,
+  matchSession,
+} from '../../../core';
+import {
+  CreateEthnologueLanguage,
+  EthnologueLanguage,
+  UpdateEthnologueLanguage,
+} from '../dto';
+
+@Injectable()
+export class EthnologueLanguageService {
+  constructor(
+    private readonly db: DatabaseService,
+    private readonly config: ConfigService,
+    @Logger('ethnologue:language:service') private readonly logger: ILogger
```

Comment:
```suggestion
    @Logger('language:ethnologue:service') private readonly logger: ILogger
```
This is a namespaced name and we are in the LanguageModule.